ill.js

```diff
@@ -0,0 +1,35 @@
+//ryanve.dev/varam inspo
```

Comment:
this code inspired me to create [varam](https://ryanve.dev/varam) to abstract the technique